src/styles/header.scss

```diff
@@ -0,0 +1,58 @@
+@import 'variables';
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+
+  .logo {
+    display: flex;
+  }
+
+  .nav__list {
+    list-style-type: none;
+    display: flex;
+    padding: 0;
+    margin: 0;
+
+    .nav__item {
+      margin-right: 20px;
+      position: relative;
+
+      &:last-child {
+        margin-right: 0;
+      }
+
+      .nav__link {
+        display: flex;
+        align-items: center;
+        height: 60px;
+        text-decoration: none;
+        color: $mainColor;
```

Comment:
Use kebab-case for variable names to maintain consistency with the code style rules. The variable `$mainColor` should be `$main-color`.